ert3/console/__init__.py

```diff
@@ -85,10 +84,9 @@ def _run_experiment(workspace_root, experiment_name):
     ert3.storage.init_experiment(workspace_root, experiment_name)
 
     coefficients = _generate_coefficients()
-    ert3.storage.add_input_data(workspace_root, experiment_name, coefficients)
-
-    response = ert3.evaluator.evaluate(coefficients)
-    ert3.storage.add_output_data(workspace_root, experiment_name, response)
+    with ert3.storage.Storage(workspace_root, experiment_name) as storage:
+        storage.input_data = coefficients
+        storage.output_data = ert3.evaluator.evaluate(coefficients)
```

Comment:
Is the idea of the context manager that it will take care of open/close operations related to storage, for example connections in the case of a database? If so it would be interesting to to open the current storage file when you enter the context, and close it when you exit, just to "stay true" to the concept 🙂

Reply:
Yes, that was the idea, so it would be easier to abstract. Not very clear in this context, but this is why I wrote it leaves something to be desired ;)